src/main/java/pl/javastart/task/Televisor.java

```diff
@@ -0,0 +1,21 @@
+package pl.javastart.task;
+
+public class Televisor {
+    boolean state;
+
+    void turnOn() {
+        state = true;
+    }
+
+    void turnOff() {
+        state = false;
+    }
+
+    void showStatus() {
+        System.out.printf("Telewizor jest włączony: %b\n", state);
+    }
+
+    public Televisor(boolean state) {
+        this.state = state;
+    }
```

Comment:
konstruktor najczęściej umieszcza się zaraz po polach, kolejność to: pola, konstruktor, metody